Add zip functionality (#825)
box/box-java-sdk: doc/zip_download.md

Zip Download
======

> Allows you to create a zip file of Box files and folders and download them.

Reviewer: ```suggestion
Allows you to create a temporary zip file of Box files and folders and download them.
```

<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->

- [Create a Zip File](#create-a-zip-file)
- [Download a Zip File](#download-a-zip-file)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

Create a Zip File
---------------

Calling [`BoxZip.create(String name, List<JsonObject> items)`][create-a-zip-file] will let you create a new zip file with the specified name and 

> with the specified items and will return a `BoxZipInfo` object with the download link. This file does not show up in your Box account, but will be available for 

Reviewer: ```suggestion
with the specified items and will return a `BoxZipInfo` object with the download link. This file does not show up in your Box account, but will be temporarily 
```

> for download.

Reviewer: ```suggestion
available for download.
```
